taurus_pyqtgraph/taurusmodelchoosertool.py

```diff
@@ -29,14 +29,17 @@
 
 __all__ = ["TaurusModelChooserTool", "TaurusImgModelChooserTool"]
 
-from taurus.external.qt import QtGui
+from taurus.external.qt import Qt, QtGui
```

Comment:
Since you are importing Qt, importing QtGui seems redundant. I suggest to replace QtGui usages by Qt